### PR TITLE
Code and example fixes before publishing.

### PR DESCRIPTION
**Pull request by filipstachura**

### CODE REVIEW

File: R/router.R [hunk 1]
```diff
@@ -7,6 +7,10 @@ HIDDEN_ROUTE_INPUT <- '_location'
   shiny::addResourcePath('shinyrouter', system.file('www', package = 'shinyrouter', mustWork = TRUE))
 }
 
+#' Escapes routing path from not safe characters.
```
Comment (KrzysztofWrobel) [inline R/router.R]: This should state it's internal function.

Comment (KrzysztofWrobel) [inline R/router.R]: THis should be stated in all internal functions

File: R/router.R [hunk 2]
```diff
@@ -69,7 +81,14 @@ create_router_callback <- function(root, routes) {
 
 #' Creates router.
```
Comment (KrzysztofWrobel) [inline R/router.R]: Add somthing like: This has to be called in main body

File: R/router.R [hunk 3]
```diff
@@ -79,7 +98,7 @@ make_router <- function(default, ...) {
 
 #' Creates an output for router. This configures client side.
```
Comment (KrzysztofWrobel) [inline R/router.R]: This has to be called in  which moment.... ?

File: examples/basic/app.R [hunk 4]
```diff
@@ -1,71 +1,42 @@
...
-                                                  uiicon("heart outline like"), "17 likes"),
-                          uiicon("comment"), "3 comments"),
-                      div(class = "extra content", div(class = "ui large transparent left icon input",
-                                                       uiicon("heart ouline"), tags$input(type = "text",
-                                                                                          placeholder = "Add Comment..."))))
-              )
-          )
+    style = "margin-top: 1em",
+    div(class = "ui grid",
+      div(class = "four wide column",
```
Comment (KrzysztofWrobel) [inline examples/basic/app.R]: Comment that this is Semantic specific classes

File: R/router.R (hunk 1 above)
Comment (filipstachura) [inline R/router.R]: done

File: R/router.R (hunk 2 above)
Comment (filipstachura) [inline R/router.R]: done

File: R/router.R (hunk 3 above)
Comment (filipstachura) [inline R/router.R]: done

File: examples/basic/app.R (hunk 4 above)
Comment (filipstachura) [inline examples/basic/app.R]: done

**Pull request merged**